packages/block-editor/src/components/block-settings-menu-controls/index.js

```diff
@@ -31,9 +36,14 @@ const BlockSettingsMenuControlsSlot = ( { fillProps, clientIds = null } ) => {
 
 	return (
 		<Slot fillProps={ { ...fillProps, selectedBlocks } }>
-			{ ( fills ) =>
-				! isEmpty( fills ) && <MenuGroup>{ fills }</MenuGroup>
-			}
+			{ ( fills ) => {
+				return (
+					<MenuGroup>
+						{ ! isEmpty( fills ) && fills }
```

Comment:
It's engough to have now:
```suggestion
						{ fills }
```

This check existed to avoid rendering the menu group with no content inside.